index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+A code review process involves having one developer review another developers changes to the source code before it is submitted. Doing this effectively can spot defects and share knowledge.
+
+**Having a code review process will simultaneously upskill your developers and improve the quality of your code.**
+
+Code review _ensures that the code understood by everyone_, as well as sharing coding best practices organically.
+
+This seminar will cover all the advantages of a code review process. It will explain how to implement one, and then how to make the most of it.
+
+[Get in touch](mailto:hi@tkiley.co.uk) today to learn about how code review can help your software development practices. This seminar can be customised to include reviewing someones code as part of a demonstration.
+
+#### Seminar Syllabus
+
+The seminar can cover:
+
+ * What a code review process is.
+ * What to look for in a code review.
+ * How to make your code easy to review.
+
+Software & Technologies:
+
+ * [Gitlab](https://about.gitlab.com/){:target="_blank"} - a widely used cloud tool for conducting code reviews
+
+As with all the other seminars, get in touch if you use a different platform for conducting code reviews and I can tailor the course to use that website to make it easier to apply this to your existing workflows.
+
+### Continuous Integration
+
+Continuous Integration (CI) is an industry standard approach to automatically check changes to projects automatically. This catches problems before they affect other developers and customers.
+
+**Continuous Integration allows developers to make changes confidently and efficiently.**
+
+Using Continuous Integration (CI) technologies is a cheap and easy way to increase the reliability of your software. Every change to the code is checked automatically, freeing experts up from testing changes by hand and to focus on their expertise. It helps developers work together, without breaking each others work.
+
+This seminar will cover setting up a basic continuous integration system, and go into details of what a good CI system should do.
+
+#### Seminar Syllabus
+
+The seminar can cover:
+
+ * What CI is and does.
+ * How to set up a basic CI.
+ * How to use a CI system.
+ * Maximizing the value of a CI system.
+
+Software & Technologies:
+
+ * [Gitlab CI/CD](https://docs.gitlab.com/ee/ci/){:target="_blank"} - a popular cloud platform for running a CI system on
+ * [Python unittest](https://docs.python.org/3/library/unittest.html){:target="_blank"} - the built-in unit testing framework in Python
+
+This seminar can be adjusted to use examples from your preferred technologies. After the course I'd be happy to help with troubleshooting any problems that arise.
+
+## Services
+
+I offer an off-the-shelf course, with the option to tailor any and all of the seminars to better meet your precise needs and technologies.
```

Comment:
Again I wouldn't mention "off-the-shelf". e.g. "I offer a course covering the most important aspects of software development best practices, with the option..."